garrysmod/addons/[brick]bricks_essentials/lua/bricks_server/modules/essentials/submodules/crafting/server/sv_crafting_system.lua

```
@@ -305,24 +315,46 @@ function BRICKS_SERVER.Func.RespawnGarbage()
 		local angles = Angle( tableString[4], tableString[5], tableString[6] )
 
 		if( position and angles ) then
-			local nearbyEnts = ents.FindInSphere( position, 10 )
+			local garbageType = ""
+			if( BRICKS_SERVER.CONFIG.CRAFTING.GarbageTypes ) then
```

Luacheck (1.2.0)

[warning] accessing undefined variable 'BRICKS_SERVER'

(W113)